lib/Subscribers/LinkTokens.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace MailPoet\Subscribers;
+
+use MailPoet\Models\Subscriber;
+
+class LinkTokens {
+
+  const TOKEN_LENGTH = 32;
+
+  function generateToken($email) {
```

Comment:
I was a bit confused by those two names `generateToken` and `getToken`, which is for what. It seems to me that `generateToken` should be private. That would make the interface clearer and more obvious which method to use. What do you think?